2013/data_file_specification.json

```diff
@@ -260,6 +390,9 @@
                "end": "54",
                "length": "5",
                "dataType": "N",
+               "format": "currency,thousands",
```

Comment:
An object makes sense too, but in this case would you need the key? Wouldn't you just loop through the array and apply the appropriate format? Is there a case where you would want to single one out based on the key?

Either works though.